src/StripeTests/Entities/StripeTokenTest.cs

```diff
@@ -0,0 +1,24 @@
+namespace StripeTests
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+
+    using Newtonsoft.Json;
+    using Stripe;
+    using Xunit;
+
+    public class StripeTokenTest : BaseStripeTest
+    {
+        [Fact]
+        public void Deserialize()
+        {
+            string json = GetFixture("/v1/tokens/tok_123");
```

Comment:
Part of me wants to test each type of token. I know you might say that's not what this test suite should be about but it's the last time I'll raise it since it's the last library we migrate :)